recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@ package:
   version: 1.2
```

Comment:
Since this is a change to how the Bundle is put together, it should probably be a bump to the version number:

Maybe 1.3.0 ?